core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerRpcClient.java

```diff
@@ -100,7 +100,10 @@
      * unrelated services.
      *
      * @param factory a function that takes a host+method and creates a new {@link CircuitBreaker}
+     *
+     * @deprecated Use newDecorator(), building a CircuitBreakerMapping using CircuitBreakerMapping.Builder().
```

Comment:
```suggestion
     * @deprecated Use {@link #newDecorator(CircuitBreakerMapping, CircuitBreakerRuleWithContent)} with
     *             {@link CircuitBreakerMapping#perHostAndMethod(BiFunction)}.
```
Could you also fix the Javadoc in `CircuitBreakerClient`, please? 😄

Reply:
Updated, thanks!